doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2751,10 +2751,10 @@ Peripheral samples
 
 .. rst-class:: v2-8-0 v2-7-0
 
-NCSDK-30284: The :ref:`radio_test` sample only transmits the first burst of data in duty-cycle mode on the 4 Mbit data rate on the nRF54L15 SoC
+NCSDK-30284: The :ref:`radio_test` sample only transmits the first burst of data in duty-cycle mode on the 4 Mbit data rate.
```

Comment:
```suggestion
NCSDK-30284: The :ref:`radio_test` sample only transmits the first burst of data in duty-cycle mode on the 4 Mbit data rate
```
First line always without a full stop in the issue description with the Jira issue.

Reply:
Thanks, I removed the full stop